extensions/norwegian-public-transport/src/Search/Actions.tsx

```diff
@@ -0,0 +1,51 @@
+import { Action, ActionPanel, Icon, Image } from "@raycast/api";
+import { getFavicon } from "@raycast/utils";
+import { Feature } from "../types";
+
+type ActionsProps = {
+  venue: Feature;
+  isFavorite: boolean;
+  onAction: () => void;
+  onSave: () => void;
+};
+
+export function Actions({ venue, isFavorite, onAction, onSave }: ActionsProps) {
+  return (
+    <ActionPanel>
+      <Action title="View Departures" icon={Icon.ArrowRight} onAction={onAction} />
+      <Action
+        title={
+          isFavorite
+            ? `Remove ${venue.properties.name} From Favorites`
+            : `Add ${venue.properties.name} to Favorites`
+        }
+        icon={isFavorite ? Icon.StarDisabled : Icon.Star}
+        shortcut={
+          isFavorite ? { modifiers: ["cmd", "shift"], key: "s" } : { modifiers: ["cmd"], key: "s" }
+        }
```

Comment:
```suggestion
        shortcut={Keyboard.Shortcut.Common.Pin}
```
Maybe try to use the same hotkeys as Raycast does, you can find them in `Keyboard`

Reply:
Oh, sweet! I didn't realize there are defaults for this 💯